server/services/networkProviderExtension.go

```diff
@@ -23,13 +24,21 @@ func (extension *networkProviderExtension) valueToNativeAmount(value string) *ty
 }
 
 func (extension *networkProviderExtension) valueToCustomAmount(value string, currencySymbol string) *types.Amount {
+	currency, ok := extension.provider.GetCustomCurrencyBySymbol(currencySymbol)
+	if !ok {
+		log.Warn("valueToCustomAmount(): unknown currency", "symbol", currencySymbol)
```

Comment:
Actually, this should not happen in practice - since `valueToCustomAmount` is only called for handled operations. Operations involving unknown tokens are simply ignored.